src/pages/student/quizzes/QuizResult/index.js

```diff
@@ -29,149 +30,101 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
   return (
     <div>
       {viewResults == false ? (
-        <Container>
-          <div className={style.Resulttopic}>
-            <p className={style.Resultparagraph}>
-              <b>Topic:</b> {title}
-            </p>
-            <Badge bg='light' className={style.Resulttml}>
-              <div>
-                <p className={style.spanForTimeLeft}>Time Left:</p>
-              </div>
-              <Badge bg='secondary' className={style.Resulttimer}>
-                00:00
-              </Badge>
-            </Badge>
-          </div>
-          <Card.Body className={style.ResultwholeBodyCard}>
-            <div className={style.ResultupperBodyCard}>
-              <div className={style.ResultcardLeft}>
-                <div className={style.ResultscoreDisplay}>
-                  <Card.Title className={style.ResultquizPraise}>
-                    Great Job!
-                  </Card.Title>
-                  <Card.Text className={style.Resultscore}>
-                    {' '}
-                    <span>
-                      <b>{score}</b>
-                    </span>
-                    <b>/{total}</b>{' '}
-                  </Card.Text>
+        <Container className={style.card}>
+          <div>
+            <div className={style.Resulttopic}>{title}</div>
+            <Card.Body className={style.ResultwholeBodyCard}>
+              <div className={style.ResultupperBodyCard}>
+                <div className={style.ResultcardLeft}>
+                  <div className={style.ResultscoreDisplay}>
+                    <div className={style.ResultquizPraise}>Great Job!</div>
+                    <div className={style.ResultquizRemarks}>You Passed</div>
+                    <Card.Text className={style.Resultscore}>
+                      {' '}
+                      <span
+                        className={
+                          score < passing ? `${style.fail}` : `${style.pass}`
+                        }
+                      >
+                        <b>{score}</b>
+                      </span>
+                      <b>/{total}</b>{' '}
+                    </Card.Text>
+                  </div>
                 </div>
+                <Card className={style.Resultcard2}>
+                  <Card.Img variant="top" />
+                  <Card.Body width="100px" style={{ margin: '5px' }}>
+                    <center>
+                      <Card.Title>Friend&apos;s Scores</Card.Title> <hr />
+                    </center>
+                    <Card.Text className={style.ResultfscoreCardText}>
+                      <div className={style.ResultfriendScore}>
```

Comment:
- [ ] You can use table element for this with three columns: image, name, and score. Set text alignment to right for the score column.